Update 1508916185.rst (#76)
TYPO3-Documentation/TYPO3CMS-Exceptions: Documentation/Exceptions/1508916185.rst

..
   TYPO3 V9 - 19/08/2022
   ==================================

   Installation Overview
   ---------------------

   TYPO3 V9.5.31
   
   The Issue
   ---------
   

>    After migration from V8 to V9, this message is displayed when trying to show the site :

Reviewer: ```suggestion
   After migration from V8 to V9, this message is displayed when trying to show the site:
```

   Oops, an error occurred!
   Invalid config.metaCharset: Unknown charset "UTF-8"
   More information regarding this error might be available online.   

   Solution
   --------

>    Check if the config.metaCharset is the right one and not in uppercase UTF-8 instead of utf-8, it should be like this :

Reviewer: ```suggestion
   Check if :typoscript:`config.metaCharset` is correct and not in uppercase UTF-8 instead of lowercase utf-8, it should be like this :
```

   
   config {
      metaCharset = utf-8

>       renderCharset = utf-8

Reviewer: `renderCharset` was removed with TYPO3 v8.0, so it should not be mentioned here.
See: https://docs.typo3.org/c/typo3/cms-core/main/en-us/Changelog/8.0/Breaking-73794-RenderCharsetOptionRemoved.html

>       renderCharset = utf-8

Reviewer: ```suggestion
```
